doc/nrf/known_issues.rst

```diff
@@ -19,7 +19,7 @@ Use the drop-down filter to see known issues for previous releases and check if
 
    <select name="versions" id="versions-select">
      <option value="all">All versions</option>
-     <option value="v1-4-99-dev1">v1.4.99-dev1</option>
+     <option value="v1-5-99">v1.5.99</option>
```

Comment:
> I can't comment on the links.txt file since it doesn't show up as changed (it will after Pekka's PR is merged), but please use [www.segger.com](http://www.segger.com) and not just segger.com (changed on master already).
> 
Fixed!
> Also, need to "unorphan" the change log and known issues, but that can be done only after the release notes PR is merged.
Also fixed :)

Reply:
> This should be 1-5-0, not 1-5-99 (because we only have known issues for releases, not for master).
> That change should be on master already, so no need to update this file.

Rebased and fixed